### DISCUSSION TITLE
Account Suspended

### DISCUSSION THREAD

**Discussion by khorshedHasanTutul** [community/community#102834] [Other Feature Feedback, Questions, & Ideas] [answered]
### Account Suspended

Hi github community
One of my colleagues github account has been suspended without any reason.They dont even sent any mail for warnings or rule violation.

His account details 
**UserName : hmuzzal**
[redacted due to PII]
Ticket ID: 2565712
please help about to resolve that issue.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @khorshedHasanTutul ,

I've removed your private information from your post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
